go/go.work

```diff
@@ -0,0 +1,3 @@
+go 1.22.5
```

Comment:
We try to keep the exact Go version in sync across a few projects, so would be good to keep this at `1.22.2` so that we can upgrade all projects at the same time. 

```suggestion
go 1.22.2
```

Reply:
sure  👍